tmt/steps/provision/podman.py

```diff
@@ -138,13 +140,16 @@ def start(self):
             '-v', f'{tmt_workdir}:{tmt_workdir}:Z', '-itd', self.image],
             message=f"Start container '{self.image}'.")[0].strip()
 
-    def ansible(self, playbook):
+    def ansible(self, playbook, inventory=None, options=None):
         """ Prepare container using ansible playbook """
         playbook = self._ansible_playbook_path(playbook)
+        inventory = inventory or f'{self.container},'
+        options = f'{options} ' or ''
+        local_temp = os.path.expanduser('~/.ansible/tmp')
```

Comment:
What is this used for? Probably add a comment?

Reply:
seems like a leftover.